api/v2alpha1/pipeline_spec.proto

```diff
@@ -367,6 +367,21 @@ message PipelineTaskSpec {
   // Reference to a component.  Use this field to define either a DAG or an
   // executor.
   ComponentRef component_ref = 7;
+
+  // Trigger policy defines how the task gets triggered. If a task is not
+  // triggered, it will run into SKIPPED state.

```

Comment:
I forgot to fix the comment here. We changed the state to be NOT_TRIGGERED to be backward compatible. SKIPPED state is still preserved for caching purpose. Could you change the state here to NOT_TRIGGERED and i will send another CL to fix internal proto comments.

Reply:
Done.